syntax = "proto3";

message Request {
    string Value = 1;
}

message Response {
    string Value = 1;
}

service TestService {
    rpc ServerSide(Request) returns (stream Response) {}
    rpc ClientSide(stream Request) returns (Response) {}
    rpc Bidirectional(stream Request) returns (stream Response) {}
}
